wrappers/binaries.go

```diff
@@ -22,11 +22,15 @@ package wrappers
 
 import (
 	"os"
+	"path/filepath"
 
 	"github.com/snapcore/snapd/dirs"
+	"github.com/snapcore/snapd/osutil"
 	"github.com/snapcore/snapd/snap"
 )
 
+var completeSh = filepath.Join(dirs.SnapMountDir, "core", "current", "usr", "lib", "snapd", "complete.sh")
```

Comment:
about it living in dirs, maybe?
but yes this needs to be the "outside" one (which it is; the other one has Core in its name)